src/components/forms/DataTypes.ts

```diff
@@ -0,0 +1,48 @@
+import {SelectOptionWithKeyNameAndAbbreviation} from './SelectOptionInterface';
+
+
+export class DataTypes {
+    static CITE: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'CITE',
+        name: 'CITE-seq'
+    };
+    static HYB: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'HYB',
+        name: 'Hybrid Capture'
+    };
+    static RNA: SelectOptionWithKeyNameAndAbbreviation = {
+        key:'RNA',
+        name: 'RNA-Seq'
+    }
+    static SCRNA: SelectOptionWithKeyNameAndAbbreviation = {
+        key:'scRNA',
+        name: 'scRNA-Seq'
+    }
+    static SPT: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'SPT',
+        name: 'Spatial Transcriptomics'
+    }
+    static SNRNA: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'snRNA',
+        name: 'snRNA-Seq'
+    }
+    static WGS: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'WGS',
+        abbreviation:'WGS',
+        name: 'Whole Genome'
+    }
+    static WES: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'WES',
+        abbreviation: 'WES',
+        name: 'Whole Exome'
+    }
+
+    static dataTypesList:SelectOptionWithKeyNameAndAbbreviation[] = [DataTypes.CITE,
```

Comment:
This field is called `nihInstitutesAndCentersList` in `NIHInstitutesAndCenters`. If the fields have a similar meaning, maybe a generate name could be used so the name is consistent in all types. I don't think including the class name as part of the field name is necessary, since all uses of the field outside its definition must be explicitly scoped. What about `values` (or `VALUES` since it's a `static` member)?

Reply:
I like the thought.  VALUE it is!